.github/workflows/stable-merge-check.yml

```
@@ -17,8 +18,25 @@ jobs:
   # Job to handle /integration-tests-ok command
   process-integration-command:
     runs-on: ubuntu-latest
-    if: ${{ github.event.issue.pull_request && contains(github.event.comment.body, '/integration-tests-ok') }}
+    if: contains(github.event.comment.body, '/integration-tests')
     steps:
+      - name: Debug Event Context
+        run: |
```

actionlint (1.7.8)

"github.event.comment.body" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)